demo/summer-2016.html

```diff
@@ -240,14 +239,35 @@
         enableLiveAutocompletion: true
     });
 
+    var HashHandler = require("ace/keyboard/hash_handler").HashHandler; 
+    keyboardHandler = new HashHandler();
+    keyboardHandler.addCommand({
+      name: 'item.',
+      bindKey: {win: 'Enter', mac: 'Enter'},
+      exec: function(editor) {
+        var pos = editor.getCursorPosition()
+        if (editor.getSession().getContext(pos.row) == "list") {
+          var currline = editor.session.getLine(pos.row);
```

Comment:
Usually programmers use "cur" prefix (one r) and capitalize the second word: `curLine`